src-java/floodlight-service/floodlight-modules/src/main/resources/floodlightkilda.properties.example

```diff
@@ -59,24 +63,31 @@ net.floodlightcontroller.topology.TopologyManager.pathMetric=latency
 net.floodlightcontroller.topology.TopologyManager.maxPathsToCompute=3
 #org.openkilda.floodlight.KildaCore.command-processor-workers-count = 4
 #org.openkilda.floodlight.KildaCore.command-processor-workers-limit = 32
-#org.openkilda.floodlight.KildaCore.command-processor-deferred-requests-limit = 8
+org.openkilda.floodlight.KildaCore.command-processor-deferred-requests-limit = 64
 #org.openkilda.floodlight.KildaCore.command-processor-idle-workers-keep-alive-seconds = 300
-#org.openkilda.floodlight.KildaCore.flow-ping-magic-src-mac-address=00:26:E1:FF:FF:FE
-org.openkilda.floodlight.KildaCore.server42-upd-port-offset=5000
 org.openkilda.floodlight.KafkaChannel.environment-naming-prefix=
 org.openkilda.floodlight.KafkaChannel.bootstrap-servers=kafka.pendev:9092
 #org.openkilda.floodlight.KafkaChannel.heart-beat-interval=1
-#org.openkilda.floodlight.kafka.KafkaMessageCollector.consumer-executors=10
-#org.openkilda.floodlight.kafka.KafkaMessageCollector.consumer-disco-executors=10
+org.openkilda.floodlight.kafka.KafkaMessageCollector.consumer-executors=20
```

Comment:
why all these are changed?

Reply:
Looks like I have grabbed all difference between existing '.example' file and produced from the template. Reverted.